src/components/stepTwo/index.js

```diff
@@ -21,7 +21,7 @@ export class stepTwo extends Component {
       tokenValues: {
         name: this.props.tokenStore.name,
         ticker: this.props.tokenStore.ticker,
-        decimals: this.props.tokenStore.decimals || 0
+        decimals: this.props.tokenStore.decimals || initialStepTwoValues.token.decimals
```

Comment:
I'd rather prefer if the change is done directly in the Store:

https://github.com/poanetwork/token-wizard/blob/1151b01502267b85b02e304ab9973f1d40a372f4/src/stores/TokenStore.js#L24

and enforce a consistency. Also, can be a good option to get rid of the `initialStepTwoValues` or use those values to setup all the initial values in the required Stores.

What you think?

Reply:
@fernandomg 
It's ok for me. I think it can be done with some refactoring. We should keep the current default values ​​so we do not generate any problems